.devcontainer/Dockerfile

```diff
@@ -12,22 +11,23 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get -y install --no-install-recommends apt-utils dialog && \
     apt-get -y install --no-install-recommends apt-transport-https ca-certificates software-properties-common libssl-dev libffi-dev \
         dotnet-sdk-3.1 build-essential gnupg-agent dnsutils httpie bash-completion curl wget git unzip jq lsb-release procps gnupg2 && \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9 && \
+    curl -O https://cdn.azul.com/zulu/bin/zulu-repo_1.0.0-2_all.deb && \
+    apt-get install ./zulu-repo_1.0.0-2_all.deb && \
+    apt-get update && \
+    apt-get -y install zulu11-jdk && \
     apt-get -y upgrade
 
 COPY .devcontainer/library-scripts /tmp/library-scripts/
 
 # [Option] Install Maven
 ARG INSTALL_MAVEN="true"
 ARG MAVEN_VERSION="3.6.3"
-RUN if [ "${INSTALL_MAVEN}" = "true" ]; then su vscode -c "source /usr/local/sdkman/bin/sdkman-init.sh && sdk install maven \"${MAVEN_VERSION}\""; fi
-
-# [Option] Install Node.js
```

Comment:
if you're removing the node installation, then you can remove the `node-debian.sh` file from the library-scripts folder.